src/index.js

```diff
@@ -8,13 +8,7 @@ const mergeFolders = require('./merge-folders').merge
 function buildBundle (config) {
   return rollup.rollup(config)
     .then(function (bundle) {
-      return bundle.write({
-        format: config.format || 'es6',
```

Comment:
@snuggs In fact I removed the "logic" because for "bundle.write" I have more options than Rollem has. For example I am using "intro" and "outro" which are the "bundle.write" options. And without removing the existing restriction in Rollem I would not be able to use this great plugin.

If you do require to have "format: es6" then just create a new object, put the property in there and then just copy all remaining properties from initial config.